docker-compose.yml

```diff
@@ -201,6 +201,7 @@ services:
       - path: ./apps/trustlab/.env
       - path: ./apps/trustlab/.env.local
         required: false
+    image: codeforafrica/trustlab-ui:latest
```

Comment:
@kilemensi Doesn't this make `make trustlab` command test an already published image when it is supposed to build an image locally? I thought docker-compose is only for dev purposes.

Reply:
[Not necessary](https://docs.docker.com/reference/compose-file/build/#using-build-and-image) @koechkevin ... we can also include [`pull_policy`](https://docs.docker.com/reference/compose-file/services/#pull_policy) to build every time.